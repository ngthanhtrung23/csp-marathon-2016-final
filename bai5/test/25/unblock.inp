1..222
1.3344
005..6
..57.6
8887.9
AAA..9
