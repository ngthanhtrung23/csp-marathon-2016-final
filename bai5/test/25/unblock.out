27
2 L 1
3 L 1
4 L 1
6 U 2
7 U 1
9 U 2
8 R 3
5 D 1
0 R 1
1 D 3
0 L 1
3 L 1
5 U 1
8 L 1
9 D 1
6 D 1
2 R 1
5 U 2
0 R 1
1 U 1
8 L 2
7 D 1
0 R 2
5 D 1
2 L 1
6 U 1
0 R 1

